sections/06_discussion.md

```diff
@@ -49,6 +49,86 @@ with only a couple GPUs.*
+network size or performance [@tag:Yasushi2016_cgbvs_dnn].
+
+Steady improvements in GPU hardware may alleviate this issue somewhat, but it
+is not clear whether it can occur quickly enough to keep up with the increasing
+amount of available biological data. Much has been done to minimize the memory
+requirements of neural networks [@tag:CudNN @tag:Caruana2014_need
+@tag:Gupta2015_prec @tag:Bengio015_prec @tag:Sa2015_buckwild
+@tag:Chen2015_hashing @tag:Hubara2016_qnn], but there is also growing
+interest in specialized hardware, such as field-programmable gate arrays
+(FPGAs) [@tag:Edwards2015_growing_pains Lacey2016_dl_fpga] and
+application-specific integrated circuits (ASICs). Specialized hardware promises
+improvements in deep learning at reduced time, energy, and memory
+[@tag:Edwards2015_growing_pains]. Logically, there is less software for highly
+specialized hardware [@tag:Lacey2016_dl_fpga], and it could be a difficult
+investment for those not solely interested in deep learning. However, it is
+likely that such options will find increased support as they become a more
+popular platform for deep learning and general computation.
+
+Distributed computing is a general solution to intense computational
+requirements, and has enabled many large-scale deep learning efforts. Early
+approaches to distributed computation were not suitable for deep learning
+[@tag:Dean2012_nips_downpour], but significant progress has been made. There
+now exist a number of algorithms [@tag:Dean2012_nips_downpour @tag:Dogwild
+@Sa2015_buckwild], tools [@tag:Moritz2015_sparknet @tag:Meng2016_mllib
+@tag:Tensorflow], and high-level libraries [@tag:Keras, @tag:Elephas] for deep
+learning in a distributed environment, and it is possible to train very complex
+networks with limited infrastructure [@tag:Coates2013_cots_hpc]. Besides
+handling very large networks, distributed or parallelized approaches offer
+other advantages, such as improved ensembling [@tag:Sun2016_ensemble] or
+accelerated hyperparameter optimization [@tag:Bergstra2011_hyper
+@tag:Bergstra2012_random].
+
+Cloud computing, which has already seen adoption in genomics
+[@tag:Schatz2010_dna_cloud], could facilitate easier sharing of the large
+datasets common to biology [@tag:Gerstein2016_scaling @tag:Stein2010_cloud],
+and may be key to scaling deep learning. Cloud computing affords researchers
+significant flexibility, and enables the use of specialized hardware (e.g.,
+FPGAs, ASICs, GPUs) without significant investment. With such flexibility, it
+could be easier to address the different challenges associated with the
+multitudinous layers and architectures available
+[@tag:Krizhevsky2014_weird_trick]. Though many are reluctant to store sensitive
+data (e.g., patient electronic health records) in the cloud,
+secure/regulation-compliant cloud services do exist [@tag:RAD2010_view_cc].
+
+*TODO: Write the transition once more of the Discussion section has been
```

Comment:
* In regards to cloud computing, it think it enables broader use in the short term. This is largely due to the number of small grants out there for cloud credit (e.g., https://aws.amazon.com/grants/), which are relatively simple to apply for. Additionally, there seems to be significant support for it (https://datascience.nih.gov/commons). Perhaps a bigger issue would be the stability of such a model; it may be unwise to predicate the advancement of deep learning on accessibility to cloud computing.
* I suspect that problems with interpretation, model sharing, and reproducibility are just as limiting as hardware. Fortunately, these should be things we can fix right now without significant hardware investment. Even without cloud resources, it should be easy for any lab to acquire a GPU (https://developer.nvidia.com/academic_gpu_seeding), and begin working on these issues immediately.
* It would seem that solving hardware challenges would lead to an acceleration of research, at least based on what we've seen in the reviewed papers.